src/main/java/com/ncc/aif/ValidateAIF.java

```diff
@@ -286,25 +300,26 @@ public static void main(String[] args) {
         }
 
         // Collect the flags parsed from the arguments
-        final boolean nistFlag = flags.contains(ArgumentFlags.NIST);
+        final Restriction restriction = !flags.contains(ArgumentFlags.NIST) ? Restriction.NONE :
+                flags.contains(ArgumentFlags.HYPO) ? Restriction.NIST_HYPOTHESIS : Restriction.NIST;
```

Comment:
I'd like to leave the chained ternaries as it's an assignment. Is there something else I could do to make it more clear? Would documenting it more help?

Reply:
Nah, just leave it as it is.  Like I said, it's pretty clear what it's intended to do even if you don't parse it out...I suppose if we added a fourth option (beyond NONE, NIST_HYPOTHESIS, and NIST) it might get unwieldy at that point.